android/tests_backend/dialogs.py

```diff
@@ -28,7 +28,7 @@ async def _close_dialog():
 
         dialog._impl.show = automated_show
 
-    def setup_info_dialog_result(self, dialog):
+    def setup_info_dialog_result(self, dialog, pre_close_test_method=None):
```

Comment:
This is a foot-gun waiting to go off. If this extension point is being added, then it needs to be added (and added *everywhere*), not just stubbed out enough to make the interpreter stop complaining. Otherwise, someone will try to use the method, assume the test is executed, and then be surprised when it isn't.